website/static/js/comment.js

```diff
@@ -14,6 +14,187 @@ var osfHelpers = require('js/osfHelpers');
 var CommentPane = require('js/commentpane');
 var markdown = require('js/markdown');
 
+var caret = require('Caret.js');
+var atWho = require('At.js');
```

Comment:
They do need to be required on the page or Webpack won't load those libraries. If it would make it more clear, I could not assign a variable and just have `require(<library>);`. What do you think?